src/main/java/mike/Parser.java

```diff
@@ -0,0 +1,137 @@
+package mike;
+
+public class Parser {
+    private final String userInput;
+    private final String[] splitString;
```

Comment:
Perhaps you can rename `splitString` to `splitStrings`, as it is an array of strings.